src/services/PostsService.ts

```diff
@@ -25,7 +25,16 @@ export class PostsService {
 		
 	public async findAll(): Promise <Posts[]> {
 		return await this.postsRepository.find();
-	}
+    }
+    
+    public async findById(id: number): Promise<Posts> {
+        const post = await this.postsRepository.findOne(id);
+        if (post) {
```

Comment:
Could be potentially simplified to:
```
if (post) return post;
throw new NotFound(...)
```

Or vice versa, as both `return` and `throw` stop program execution

Reply:
Noted and corrected.